### DISCUSSION TITLE
Fred's ImageMagick scripts: timeout

### DISCUSSION THREAD

**Discussion by snibgo** [ImageMagick/ImageMagick#4610] [Help]
For a few days, I have had problems accessing Fred's scripts. The page at http://www.fmwconcepts.com/imagemagick/index.php loads with no problem, but links from there such as http://www.fmwconcepts.com/imagemagick/adaptivegamma/index.php give the error message:

"The connection has timed out. The server at www.fmwconcepts.com is taking too long to respond."

**Comment by fmw42** [comment]
@snibgo.  I am not having any issues accessing those links/pages.  Perhaps there is an issue with an Internet server. Try a traceroute and see if there is a hang-up.

**Reply by fmw42** [reply]
If you need any particular scripts, let me know and I will email them to you.

**Comment by snibgo** [comment]
Thanks, Fred. Can you email me statsfilt, tunnelize, warplog and xpand?

The work on "-fx" is going well. The bulk of the work is done, and now I need to test it.

**Comment by snibgo** [comment]
Thanks for the scripts, Fred. "cameradeblur" has:
```
		convert \
		\( -size ${w1}x1 xc: -fx "$fxd*(i-$cx))" -scale ${w1}x${h1}! -evaluate pow 2 \) \
		\( -size 1x${h1} xc: -fx "$fyd*(j-$cy))" -scale ${w1}x${h1}! -evaluate pow 2 \) \
		-define compose:clamp=false -compose plus -composite -evaluate pow 0.5 -fx "jinc(u)" \
		$tmpF1
```
This has mismatched parentheses, such as:
```
-fx "$fxd*(i-$cx))" 
```
This expands in my test case to:
```
-fx "0.0373134*(i-134))"
```

My new "-fx" rejects this syntax.

**Reply by fmw42** [reply]
Thanks. I will fix that typo. It may explain some issues I had.
